conf/application.conf

```diff
@@ -139,5 +139,14 @@ microservice {
       environment = ""
     }
 
+    submit-uk-tax-return {
```

Comment:
is this the HIP endpoint? Does this correspond to what has been created in the stub?

Reply:
Yes, the stub needed to be updated as it didn't take the pillar2 ref as a header, using the oppunity to fix some of the semantic issues with the stub:
https://github.com/hmrc/pillar2-stubs/pull/40/files#diff-93a87e8ed9cad16136fcaa02353db7a43cb40c5ba0f4cba187dd21833533db11

Reply:
Will also raise another PR for external test